backend/src/main/resources/application-prod.yaml

```diff
@@ -1,4 +1,5 @@
 simple-report:
+  processing-mode-code: P
```

Comment:
I keep coming back to if this is the correct file to set this in for prod (and how to test it before deploying).  I don't know where we define this particular profile for prod.  The `azure-prod` profile is set in the [terraform](https://github.com/CDCgov/prime-simplereport/blob/0fc71e39f3a3fd1f011b7b47ac4b5f7a6e755979/ops/prod/api.tf#L33), but I can't find any reference to `prod`

Reply:
I was assuming since we had previously only defined the `processing-mode-code` in the stg yaml it was using the default value of "P" for all other envs... also not sure the best way to test this specifically for prod before deploying to prod either... 😬

Reply:
Covered this in eng sync, but the main [`application.yaml`](https://github.com/CDCgov/prime-simplereport/blob/4dbdd7fcb839c6fe768b45d222d7d6393afdf59d/backend/src/main/resources/application.yaml#L1-L20) groups some the profile together.  `azure-prod` applies `okta-prod` via these groups.  `prod` does exist as a profile that includes `okta-prod`, but is not applied to the app anywhere that I can find.